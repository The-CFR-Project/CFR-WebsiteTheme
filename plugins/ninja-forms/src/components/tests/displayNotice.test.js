import { render, cleanup } from '@testing-library/react';
import { DisplayNotice } from '../';

describe( 'submissions-display-notice', () => {
	afterEach( cleanup );
	it( 'Matches snapshot', () => {
		const { container } = render(
			<DisplayNotice isDismissible="false" text="Count emails sent" />
		);
		expect( container ).toMatchSnapshot();
	} );
} );
