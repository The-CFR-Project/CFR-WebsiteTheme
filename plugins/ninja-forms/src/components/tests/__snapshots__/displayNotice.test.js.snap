// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`submissions-display-notice Matches snapshot 1`] = `
<div>
  <div
    class="components-notice is-info is-dismissible"
  >
    <div
      class="components-notice__content"
    >
      Count emails sent
    </div>
    <button
      aria-label="Dismiss this notice"
      class="components-button components-notice__dismiss has-icon"
      type="button"
    >
      <svg
        aria-hidden="true"
        focusable="false"
        height="24"
        role="img"
        viewBox="0 0 24 24"
        width="24"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M13 11.8l6.1-6.3-1-1-6.1 6.2-6.1-6.2-1 1 6.1 6.3-6.5 6.7 1 1 6.5-6.6 6.5 6.6 1-1z"
        />
      </svg>
    </button>
  </div>
</div>
`;
